cim-client-sdk/src/main/java/com/crossoverjie/cim/client/sdk/io/CIMClientHandle.java

```diff
@@ -67,6 +67,7 @@ protected void channelRead0(ChannelHandlerContext ctx, Response msg) {
 
         if (msg.getCmd() != BaseCommand.PING) {
             // callback
+            //todo 线程池中配置了多个线程的话会导致消息顺序紊乱
```

Comment:
The message order should be sorted by the client using the timestamp field in the message body.

This can be implemented in a separate PR.